### PR TITLE
Improve documentation on prompt options for powerline themes

### PR DESCRIPTION
**Pull request by willprice**
I've collected the supported prompt options from reading `themes/powerline/powerline.base.bash` and documented them in all powerline* themes.

### CODE REVIEW

File: themes/powerline-plain/README.md [hunk 1]
```diff
@@ -40,14 +40,19 @@ The time/date is printed by the `date` command, so refer to its man page to chan
...
+* `last_status` - Exit status of last run command
+* `python_venv` - Python virtual environment information (`virtualenv`, `venv`
+  and `conda` supported)
+* `ruby` - Current ruby version if using `rvm`
+* `scm` - Version control information, `git` 
+* `user_info` - Current user
+* `wd` - Working directory, like `cwd` but doesn't show the full folder
+  hierarchy, only the directory you're currently in.
 
 A variables can be defined to set the order of the prompt segments:
```
Comment (nwinkler) [inline themes/powerline-plain/README.md]: While you're working on this, can you please change _variables_ to _variable_?

File: themes/powerline-naked/README.md [hunk 2]
```diff
@@ -42,14 +42,19 @@ The time/date is printed by the `date` command, so refer to its man page to chan
...
+* `last_status` - Exit status of last run command
+* `python_venv` - Python virtual environment information (`virtualenv`, `venv`
+  and `conda` supported)
+* `ruby` - Current ruby version if using `rvm`
+* `scm` - Version control information, `git` 
+* `user_info` - Current user
+* `wd` - Working directory, like `cwd` but doesn't show the full folder
+  hierarchy, only the directory you're currently in.
 
 A variables can be defined to set the order of the prompt segments:
```
Comment (nwinkler) [inline themes/powerline-naked/README.md]: While you're working on this, can you please change _variables_ to _variable_?

File: themes/powerline/README.md [hunk 3]
```diff
@@ -44,14 +44,19 @@ The time/date is printed by the `date` command, so refer to its man page to chan
...
+* `last_status` - Exit status of last run command
+* `python_venv` - Python virtual environment information (`virtualenv`, `venv`
+  and `conda` supported)
+* `ruby` - Current ruby version if using `rvm`
+* `scm` - Version control information, `git` 
+* `user_info` - Current user
+* `wd` - Working directory, like `cwd` but doesn't show the full folder
+  hierarchy, only the directory you're currently in.
 
 A variables can be defined to set the order of the prompt segments:
```
Comment (nwinkler) [inline themes/powerline/README.md]: While you're working on this, can you please change _variables_ to _variable_?

**Comment by willprice** [comment]
Done :)

**Comment by nwinkler** [comment]
Very nice - thanks!

**Pull request merged**